src/components/SpecialResolution/CreateSpecialResolutionSummary.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <div>
```

Comment:
This component should have an id. (Useful for targeting the component externally if needed.)